src/interrupt.rs

```diff
@@ -54,20 +53,18 @@ pub unsafe fn enable() {
 }
 
 /// Execute closure `f` in an interrupt-free context.
```

Comment:
Perhaps this method should now refer to critical-section, something like this:

```suggestion
/// Execute closure `f` in an interrupt-free context.
///
/// This method does not synchronise multiple cores and may disable required
/// interrupts on some platforms; see the critical-section crate for a cross-platform
/// way to enter a critical section which provides a `CriticalSection` token.
///
/// This crate provides an implementation for critical-section that simply disables
/// all interrupts which may be enabled with the `critical-section-single-core` feature.
```

Reply:
Done!